README.md

```diff
@@ -40,14 +40,14 @@ Operand Deployment Lifecycle Manager has four CRDs:
 
 ## Supported platforms
 
-You can install the Operand Deployment Lifecycle Manager on Linux® x86_64 with Red Hat® OpenShift® Container Platform version 4.2 or 4.3.
+You can install the Operand Deployment Lifecycle Manager on Linux® x86_64 with Red Hat® OpenShift® Container Platform version 4.5+.
```

Comment:
How about changing it to 4.x? Otherwise we need to update it every time OCP version is out of support.

Reply:
Probably should be 4.3+, because CRD v1 is supported in 4.3

Reply:
sure, maybe change to 4.3+. but.. why CRD v1 still supports OCP versions that are deprecated by Redhat？

Reply:
It's because RedHat support lifecycle doesn't come with k8s updates. An OCP version isn't supported by Redhat, which doesn't mean ODLM can work on it. From an open-source software perspective, ODLM can still support that.